### PR TITLE
Adds support for expiring images after N days

### PR DESCRIPTION
**Pull request by zsims**
This setting defaults to 30 and ensures "cached" build images will cycle through security updates at least every 30 days.

### CODE REVIEW

File: tests/run.bats [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
```
Comment (72636c) [inline tests/run.bats:1]: 🎉

File: tests/run.bats [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  run "${pre_command_hook}"
+
+  assert_success
+  assert_output --partial "logging in to docker"
+  assert_output --partial "pulled image"
+  assert_output --partial "looked up repository"
+  assert_output --partial "put lifecycle policy"
+
+  unstub aws
+  unstub docker
```
Comment (72636c) [inline tests/run.bats:37]: Are we missing
```suggestion
  unstub docker
  unstub sha1sum
```

File: README.md [hunk 3]
```diff
@@ -157,6 +174,14 @@ automatically applied to limit the repository to the last 10 pushed images.
 [ecr storage costs]: https://aws.amazon.com/ecr/pricing/
 [lifecycle policy]: https://docs.aws.amazon.com/AmazonECR/latest/userguide/LifecyclePolicies.html
 
+# Tests
```
Comment (72636c) [inline README.md]:
```suggestion
## Tests
```
😛

File: README.md [hunk 4]
```diff
@@ -13,6 +13,10 @@ as changes are made to your containers.
 An ECR repository to store the built Docker image will be created for you, if
 one doesn't already exist.
 
+This plugin sets the ECR lifecycle policy to ensure that only 10 images are stored. Additionally, images
+older than 30 days will automatically expire. Automatically expiring images ensures the cache remains somewhat 
+up-to-date with patches.
```
Comment (72636c) [inline README.md]: A bit of info duplicated between here and L170

**Review by 72636c — APPROVED** [review]

File: tests/run.bats (hunk 2 above)
Comment (zsims) [reply tests/run.bats:37]: Good pickup, yes thanks :-)

**Review by 72636c — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (zsims) [reply README.md]: Good catch, fixed

**Review by etaoins — APPROVED** [review]

**Pull request merged**